packages/aws-rfdk/lib/deadline/test/repository.test.ts

```diff
@@ -858,7 +858,7 @@ test('repository instance is created with correct installer path version', () =>
 
   // THEN
   const script = (repo.node.defaultChild as AutoScalingGroup).userData;
-  expect(script.render()).toMatch(/10\.1\.9\.2/);
+  expect(script.render()).toMatch(version.versionString);
```

Comment:
[`expect(...).toMatch(....)`](https://jestjs.io/docs/expect#tomatchregexp--string) will treat the passed in string as a regular expression. Because of this, the periods (`.`) in the version string will be treated as matching any character.

Instead, how about we use [`expect.stringContaining(...)`](https://jestjs.io/docs/expect#expectstringcontainingstring):

```suggestion
  expect(script.render()).toEqual(expect.stringContaining(version.versionString));
```

Reply:
Looks good now. resolved